rudder-web/src/main/scala/com/normation/rudder/web/services/ReportDisplayer.scala

```diff
@@ -138,71 +138,77 @@ class ReportDisplayer(
+            case None     =>  "without a configuration ID, although one is required"
+            case Some(id) => s"with configuration ID '${id.value}' that is unknown to Rudder"
           }
 
-          <p>The node sent a run (started at {lastRunDateTime.toString(dateFormat)}) {configIdmsg}. Either that configuration ID was deleted from Rudder DB or
-             the node is sending a corrupted configuration ID. You should update node policies with "rudder agent update -f", and if the problem persists,
-             try to regenerate policies with the "clear cache" action.</p>
+          <p>This node has no configuration policy assigned to it, but reports have been received for it {configIdmsg} (run started at {lastRunDateTime.toString(dateFormat)}).
+             Either this node was deleted from Rudder but still has a running agent or the node is sending a corrupted configuration ID.
+             Please run "rudder agent update -f" on the node to force a policy update and, if the problem persists,
+             force a policy regeneration with the "Clear caches" button in Administration > Settings.</p>
 
         case UnexpectedVersion(lastRunDateTime, Some(lastRunConfigInfo), lastRunExpiration, expectedConfigInfo, expectedExpiration) =>
           (
-            <p>{currentConfigId(expectedConfigInfo)} but we received a run, started at {lastRunDateTime.toString(dateFormat)}, with a different configuration ID ({expectedConfigInfo.configId.value}).</p>
-            <p>The time since generation of the new configuration is longer than the grace period, so all received reports will be mark as unexpected.
-               Please check that the node is able to get its new policies by running 'rudder agent update' on the node.</p>
+            <p>This node is sending reports from an out-of-date configuration policy ({expectedConfigInfo.configId.value}, run started at {lastRunDateTime.toString(dateFormat)}).
+               Please check that the node is able to update it's policy by running 'rudder agent update' on the node.</p>
+            <p>{currentConfigId(expectedConfigInfo)}</p>
           )
 
         case UnexpectedNoVersion(lastRunDateTime, Some(lastRunConfigInfo), lastRunExpiration, expectedConfigInfo, expectedExpiration) =>
           (
-            <p>{currentConfigId(expectedConfigInfo)} but we received a run, started at {lastRunDateTime.toString(dateFormat)}, without any configuration ID.</p>
-            <p>The time since generation of the new configuration is longer than the grace period, so all received reports will be marked as unexpected.
-               Please check that the node is able to get its new policies.</p>
+            <p>This node is sending reports without a configuration ID (run started on the node at {lastRunDateTime.toString(dateFormat)}), although one is required.</p>
+            <p>All reports below will be marked as unexpected. Please run "rudder agent update -f" on the node to force a policy update.</p>
```

Comment:
In that case, we said that we won't display at all unexpected message, but just the expected configuration elements without the compliance row. So the "All reports below will be marked as unexpected. " must be modified